liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -809,11 +816,16 @@ private void processContainerParams() throws MojoExecutionException {
                         + " dockerfile should be a valid Dockerfile");
                 }
             }
-    
+
             if (dockerRunOpts != null) {
                 setContainer(true);
                 return;
-            }    
+            }
+
+            if (dockerTimeout != 60) { // limitation: if user sets dockerTimeout to the default value container is not set
```

Comment:
To avoid inconsistencies due to this limitation, perhaps dockerTimeout should just not set the container option.

Reply:
Ok